tools/scripts/stress_tests.pl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env perl
```

Comment:
Looks like this script could be refactored to reduce duplication with the other one